### PR TITLE
feature/AT-7919-create-req-feedback-form

### PR DESCRIPTION
**Pull request by tonysumner-ccpo**
Ticket includes work for [AT-7919](https://ccpo.atlassian.net/browse/AT-7919) AND [AT-7943](https://ccpo.atlassian.net/browse/AT-7943)

### AC for AT-7919
Please stub in the following controls on src\DocumentReview\index.vue using [this FIGMA](https://www.figma.com/file/70guK77u1eu55qSDmIBm3g/DITCO-Reviewer?node-id=2%3A8046) as reference
1. Breadcrumb control (with dead links)
    1. Mocked data to use:
        1. Acquisitions (dead link)
        1. Demo Package (dead link)
        1. Requirements Checklist
1. Title: Requirements Checklist
1. Side Panel
    1. Button
        1. Text with icon
            1. Text: In review
            1. Icon: chevron_down
        1. Event:  None at this time
    1. LinkButton with Icon
        1. Text: Preview Form
        1. Icon: launch
        1. Event: 
            1. Click: Launches ./review.vue outlined in  AC 1.3 in a new tab
    1. Horizontal Line
    1. Subhead: Comments
    1. Description List
        1. Data Term
            1. Icon with border: Comment 
        1. Data Definition
            1. Text: Share your feedback or ask a question by hovering over a field and clicking the Comment icon.
    1. Horizontal Line

### AC for AT-7943
Please stub in the following controls on src\DocumentReview\index.vue using [this FIGMA ](https://www.figma.com/file/70guK77u1eu55qSDmIBm3g/DITCO-Reviewer?node-id=2%3A8046)as reference
**Review view**

- Load readonly view of components as they are being made
- Add subheads and horizontal lines as outlined below.
- All form fields are to be read only.
- Create all items in a numbered list.  Do not add the numbers to the labels.

### CODE REVIEW

File: src/components/ATATSlideoutPanel.vue [hunk 1]
```diff
@@ -57,6 +60,17 @@ export default class ATATSlideoutPanel extends Vue {
     }
   }
 
+  get isHeaderDisplayed(): boolean {
```
Comment (ericyoungquist-ccpo) [inline src/components/ATATSlideoutPanel.vue:63]: Remove unused method

File: src/components/ATATSlideoutPanel.vue [hunk 2]
```diff
@@ -48,6 +48,9 @@ import SlideoutPanel from "@/store/slideoutPanel/index";
 
 export default class ATATSlideoutPanel extends Vue {
   @Prop({ default: "380" }) private panelWidth!: string;
+  @Prop({ default: false }) private alwaysOpen!: boolean;
+  @Prop({ default: true }) private showHeader!: boolean;
```
Comment (ericyoungquist-ccpo) [inline src/components/ATATSlideoutPanel.vue:52]: Don't need this prop - just don't pass a `title` prop to ATATSlideoutPanel component if don't want to show the "header"/title area

File: src/components/ATATTopNavBar.vue [hunk 3]
```diff
@@ -149,7 +150,10 @@ export default class ATATTopNavBar extends Vue {
         // open external URL in new tab
         window.open(item.externalUrl, "_blank");
       }
-    }
+
+      //closes slideoutpanel if necesssary
+      SlideoutPanel.closeSlideoutPanel();
```
Comment (ericyoungquist-ccpo) [inline src/components/ATATTopNavBar.vue:155]: I believe this is already being handled after my latest branch - on navigation, slideout drawer is closed automatically now

File: src/components/icons/ATATSVGIcon.vue [hunk 4]
```diff
@@ -13,12 +13,14 @@ import {stringObj } from "types/Global";
 import Vue from "vue";
 
 import { Component, Prop, PropSync } from "vue-property-decorator";
+import ArrowLeft from "@/components/icons/ArrowLeft.vue"
```
Comment (ericyoungquist-ccpo) [inline src/components/icons/ATATSVGIcon.vue:16]: Same icon is in PR 1017 for AT-7881 - I named it `ArrowBack` -- we should probably only keep one in the codebase

File: src/components/ATATSlideoutPanel.vue (hunk 2 above)
Comment (tonysumner-ccpo) [reply src/components/ATATSlideoutPanel.vue:52]: removed

Comment (tonysumner-ccpo) [reply src/components/ATATSlideoutPanel.vue:52]: Done

File: src/components/ATATSlideoutPanel.vue (hunk 1 above)
Comment (tonysumner-ccpo) [reply src/components/ATATSlideoutPanel.vue:63]: removed

File: src/components/ATATTopNavBar.vue (hunk 3 above)
Comment (tonysumner-ccpo) [reply src/components/ATATTopNavBar.vue:156]: removed

**Review by DGabriel-CCPO — APPROVED** [review]

**Pull request merged**